pluto/umbrel-app.yml

```diff
@@ -0,0 +1,23 @@
+manifestVersion: 1
```

Comment:
If you end up needing to keep the pre-start hook then this version needs to be `1.1`. If the hook is removed then this can stay as `1`